recipes/wbplot/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "wbplot" %}
+{% set version = "1.0.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dac3c34077c4f351bfa5d61554374afce17fb6d39c9337f16c2fef633c49d660
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - matplotlib-base
+    - nibabel
+    - numpy
+    - pillow
+    - python >=3.7
+
+test:
+  imports:
+    - wbplot
+    - wbplot.utils
+
+about:
+  home: https://github.com/jbburt/wbplot
+  license: GPL-3.0-or-later
+  license_family: GPL3
+  license_file: LICENSE.txt
+  summary: "A package for automated plotting of neuroimaging maps using Connectome Workbench."
+  doc_url: 
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - jbburt
```

Comment:
@SammedAdmuthe you should add yourself or get this user permission to added them here.

Reply:
I have added myself as a recipe-maintainer